rest/route/user.go

```diff
@@ -445,7 +445,9 @@ func (h *userPermissionsGetHandler) Run(ctx context.Context) gimlet.Responder {
 	if u == nil {
 		return gimlet.NewJSONErrorResponse(errors.New("user not found"))
 	}
-	permissions, err := rolemanager.PermissionSummaryForRoles(ctx, u.Roles(), h.rm)
+	rolesToSearch, _ := utility.StringSliceSymmetricDifference(u.SystemRoles, evergreen.BasicAccessRoles)
```

Comment:
I'm a bit confused about how this works because I couldn't find where u.SystemRoles is populated. 
Will u.SystemRoles contain every single available role?

Reply:
Does that also help answer the second question? This function returns u.SystemRoles minus the basic roles.

Reply:
Nope it doesn't: you can read what happens [here](https://github.com/evergreen-ci/evergreen/blob/5b01d31f9ab5d44c2a959ac45741efcc77aeae12/vendor/github.com/evergreen-ci/utility/slice.go#L40). It returns two lists: the first is everything that is in A that is not in B, and the second is everything that is in B that is not in A. Here's the case you're describing: https://play.golang.org/p/tm0LVDrr5Zm